### PR TITLE
Update `ru_ru.json`

### PR DESCRIPTION
**Pull request by mpustovoi**
Migrated my translations from previous `main` branch Added missing lines

### CODE REVIEW

**Comment by Snownee** [comment]
please do not change the indentation

**Comment by mpustovoi** [comment]
I'm sorry
VS Code again

File: src/main/resources/assets/jade/lang/ru_ru.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	"gui.jade.by": "Сделано с %s от Snownee",
 	"gui.jade.by.hovered": "Посмотрите на все мои работы!",
 	"gui.jade.translated_by": "Русская локализация от %s",
-	"gui.jade.translators": "DrHesperus, Altegar, Heimdallr-1, TheAnaxMan, mpustovoi",
+	"gui.jade.translators": "27rogi, Altegar, TheAnaxMan, Heimdallr-1, mpustovoi",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ru_ru.json:20]: I don't see why you changed it

Comment (mpustovoi) [reply src/main/resources/assets/jade/lang/ru_ru.json:20]: I copied it from 1.20.3 branch. Tried to restore, but GitHub showed me wrong thing. And now it happens again.

File: src/main/resources/assets/jade/lang/ru_ru.json [hunk 2]
```diff
@@ -111,6 +125,7 @@
 	"config.jade.danger_zone": "Опасная зона",
 	"config.jade.reset_settings": "Сбросить настройки",
 	"config.jade.reset_settings.confirm": "Вы уверены, что хотите сбросить настройки?",
+	"config.jade.plugin_jade": "Ядро Jade",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/ru_ru.json]: this text now is "Core Features" instead of "Jade Core"

**Review by Snownee — CHANGES_REQUESTED** [review]

**Comment by Snownee** [comment]
looks good to me now, thank you

**Pull request merged**